manifests/bootstrap_rpm.pp

```diff
@@ -0,0 +1,61 @@
+# This class builds an RPM containing the bootstrap for a subscription-manager consumer
+# This file is placed in $katello_www_pub_dir.
+# @api private
+class foreman_proxy_content::bootstrap_rpm (
+  $hostname                          = $facts['networking']['fqdn'],
```

Comment:
Can we use data types for the parameters to figure out what they are?

Also, I don't think we need to align parameters. I've stopped doing that because it quickly becomes too wide if you also add data types.

Reply:
Sure.